src/components/Select.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export const SelectWrapper = styled.select`
+    
+   height: 50px;
+   width: 535px;
+   border: 1px solid #E8E8E8;
+   border-radius: 6px;
+`;
+
+export default function Select(){
+   return(
+   <SelectWrapper>
+            <option disabled>- Selecione aqui -</option>
+            <option value="Estágio">Estágio</option>
+            <option value="Júnior">Júnior</option>
+            <option value="Trainee">Trainee</option>
+   </SelectWrapper>
```

Comment:
a lógica de criar como styled-componentss é fazê-lo reutilizavel, então quando for chamado, vai ser possivel decidir: opções, texto da label do select etc de forma dinâmica por props.

do jeito que está, é uma função determinista

Reply:
Esse daqui estou mudando , fiz o map mas esta me dando problema na tipagem do typescript , se alguem do @SouJunior/frontend quiser me ajudar a resolver agradeço pois estou tentando e procurando de tudo pra resolver